godot-core/src/builtin/projection.rs

```diff
@@ -85,6 +86,25 @@ impl Projection {
         )
     }
 
+    /// Creates a new Projection that scales a given projection to fit around
+    /// a given AABB in projection space.
+    ///
+    /// _Godot equivalent: Projection.create_fit_aabb()_
+    pub fn create_fit_aabb(aabb: Aabb) -> Self {
+        let min = aabb.position;
+        let max = aabb.position + aabb.size;
+
+        let scale = Vector3::splat(2.0) / (max - min);
+        let translate = -(max + min) / (max - min);
```

Comment:
`(max - min)` is just `aabb.size` -- no need to recalculate it, let alone twice.

Generally not sure we gain much from `min`/`max` variables. The only remaining expression is then `-(max + min)`, which is the same as `(-2.0 * aabb.position - aabb.size)`.

Maybe a small comment is enough.

```suggestion
    pub fn create_fit_aabb(aabb: Aabb) -> Self {
        let translate_unscaled = -2.0 * aabb.position - aabb.size; // -(start+end)

        let scale = Vector3::splat(2.0) / aabb.size;
        let translate = translate_unscaled / aabb.size;
```

Reply:
You're right. Makes more sense instead of `min`/`max`. :+1: